example/src/demos/htmlCluster.tsx

```diff
@@ -127,7 +127,7 @@ class HtmlCluster extends React.Component<Props, State> {
         onMove={this.onMove}
         style={mapContainerStyle}
       >
-        <Cluster ClusterMarkerFactory={this.clusterMarker}>
+        <Cluster ClusterMarkerFactory={this.clusterMarker as any}>
```

Comment:
Try this (`git apply dontcast2any.patch`): https://gist.github.com/mklopets/461f206582af8dca0e96ce7a29c56f59

Reply:
still requires a cast but a much more reasonable one than to `any`